cardDetails.json

```diff
@@ -2,15 +2,13 @@
 
     "cardDetails": [
       {
-        "name": "Your Name",
-        "profession": "Your Profession",
-        "quote": "\"Your favorite quote\"</br> - Said by",
-        "twitter": "https://twitter.com",
-        "dribbble": "https://dribbble.com",
-        "behance": "https://behance.com",
-        "github": "https://github.com",
-        "linkedin": "https://linkedin.com",
-        "email": "mailto:youremail@gmail.com"
```

Comment:
It looks like you removed the card details template to add your information. Kindly check through contributors guide and make sure to add your information accordingly. Thanks!